server/package-lock.json

```diff
@@ -0,0 +1,345 @@
+{
```

Comment:
We shouldn't have a package-lock file. Might need to readd this dependancy using yarn.